wgpu-core/src/validation.rs

```diff
@@ -1168,36 +1210,25 @@ impl Interface {
             })
     }
 
-    pub(crate) fn shader_stage_from_stage_bit(stage_bit: wgt::ShaderStages) -> naga::ShaderStage {
-        match stage_bit {
-            wgt::ShaderStages::VERTEX => naga::ShaderStage::Vertex,
-            wgt::ShaderStages::FRAGMENT => naga::ShaderStage::Fragment,
-            wgt::ShaderStages::COMPUTE => naga::ShaderStage::Compute,
-            wgt::ShaderStages::MESH => naga::ShaderStage::Mesh,
-            wgt::ShaderStages::TASK => naga::ShaderStage::Task,
-            _ => unreachable!(),
-        }
-    }
-
     pub fn check_stage(
```

Comment:
Maybe it's worth adding a link to https://www.w3.org/TR/webgpu/#abstract-opdef-validating-inter-stage-interfaces somewhere in here (I'm not sure if that's all that this code is doing, however)?

Reply:
For better or for worse, this code blends several bits of validation together. We don't have a 1-to-1 structure with how validation is presented in the spec. It's worth noting, though, I guess! Added a comment as suggested with the latest force-push ([link](https://github.com/gfx-rs/wgpu/compare/8694e44f5a014d544a9db570a86868e6316fe028..83b68275cb1fdb69aa838ccd629f911e9b05816e)).